waku/v2/protocol/waku_noise/noise_handshake_processing.nim

```diff
@@ -221,8 +221,10 @@ proc processMessagePatternPayload(hs: var HandshakeState, transportMessage: seq[
   # We decrypt the transportMessage, if any
   if reading:
     payload = hs.ss.decryptAndHash(transportMessage)
+    payload = pkcs7_unpad(payload, PaddingBlockSize)
```

Comment:
I think the padding blocksize should be specified in the Noise specification. Otherwise, how would the two ends know the padding block size?